### ISSUE TITLE
Expose the DialogOpenAsync so it can be called

### ISSUE THREAD

**Issue by T-Roth** [microsoft/botbuilder-dotnet#3725]
**Is your feature request related to a problem? Please describe.**
I need a way to open a dialog in slack from the adapter

**Describe the solution you'd like**
Expose the `SlackClientWrapper.DialogOpenAsync` method on the SlackAdapter so I can open a dialog in Slack

[enhancement]

**Comment by axelsrz** [comment]
@garypretty could you comment on this one?

**Comment by garypretty** [comment]
@T-Roth Fix merged into master for next release via PR #3744.  We have now exposed the SlackClientWrapper as a public property on the adapter, so you can do something like....

```cs
var slackAdapter = (SlackAdapter)turnContext.Adapter;
var response = await slackAdapter.SlackClientWrapper.OpenDialogAsync......
```

This will be included in the nightly builds from tomorrow. Thanks for reporting.
